phpstan-baseline.neon

```diff
@@ -19709,3 +19704,43 @@ parameters:
 			count: 1
 			path: modules/social_features/social_group/src/Plugin/views/filter/FilterByGroup.php
 
+		-
```

Comment:
@nechai We should try to fix all possible PHPStan errors and not add new ones to the baseline file.

Reply:
@Ressinel Fixed as much as possible.